### DISCUSSION TITLE
How to export logs,metric and trace to Stackdriver?

### DISCUSSION THREAD

**Discussion by itsprakash84** [open-telemetry/opentelemetry-java-instrumentation#3162] [Q&A]
Hi ,

I have implemented an POC on exporting Traces, Metric to Jaeger and Prometheus. 

As I wanted to move all application logs,Trace and metric  to stack driver, so please provide me any example project or configurations.

Thanks,
Sathya Prakash

**Comment by trask** [comment]
hey @itsprakash84, this project doesn't maintain any vendor specific exporters or examples

**Reply by itsprakash84** [reply]
Ok, can u please point me to the which agent version supports and java opts. I am using below for jaeger, i hope similarly there should be one for stackdriver.
-Dotel.traces.exporter=jaeger \
  -Dotel.exporter.jaeger.endpoint=http://tempo:14250 \

**Reply by trask** [reply]
@itsprakash84 check out https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/main/exporters/auto/README.md
